.github/workflows/tasklist-ci-build-reusable.yml

```diff
@@ -104,26 +105,18 @@ jobs:
           password: ${{ steps.secrets.outputs.NEXUS_PSW }}
 
       #########################################################################
-      # Build and tag Docker images
-      - name: Docker meta
-        id: meta
-        uses: docker/metadata-action@v5
+      # Build Docker image
+      - name: Build Docker image
+        uses: ./.github/actions/build-platform-docker
```

Comment:
❓ Here the same 🤔

Reply:
as I said above: the comment is not correct
this just builds the tags literal that is used as input of build-push-action